CHANGELOG/CHANGELOG-0.3.md

```diff
@@ -7,6 +7,8 @@ Changes since `v0.3.0`:
 - Fix a bug that the validation webhook doesn't validate the queue name set as a label when creating MPIJob. #711
 - Fix a bug that updates a queue name in workloads with an empty value when using framework jobs that use batch/job internally, such as MPIJob. #713
 - Fix a bug in which borrowed values are set to a non-zero value even though the ClusterQueue doesn't belong to a cohort. #759 
+- Fix: Enforce borrowed=0 if ClusterQueue doesn't belong to a cohort. #761
```

Comment:
```suggestion
```

This is the same as Line 9 :)

Reply:
oh hehe, 761 should be the correct PR though.
Next time we won't have a mixture of manually added entries and those generated from release notes :)

Reply:
> oh hehe, 761 should be the correct PR though.
Next time we won't have a mixture of manually added entries and those generated from release notes :)

You're right.
That sounds good!